quests/6-asset-interop/README.md

```diff
@@ -29,11 +29,11 @@ contract deployed by the `Parent_Account`.**
 - [TL;DR](#tldr)
 - [How to Play](#how-to-play)
 - [The Task at Hand](#the-task-at-hand)
-  - [Stellar Assets as Soroban Tokens](#stellar-assets-as-soroban-tokens)
-  - [The Stellar Asset Contract](#the-stellar-asset-contract)
-  - [Yeah, but How do I Use That Asset Contract?](#yeah-but-how-do-i-use-that-asset-contract)
-  - [Native XLM on Soroban](#native-xlm-on-soroban)
-  - [Back to Your Quest](#back-to-your-quest)
+    - [Stellar Assets as Soroban Tokens](#stellar-assets-as-soroban-tokens)
+    - [The Stellar Asset Contract](#the-stellar-asset-contract)
+    - [Yeah, but How do I Use That Asset Contract?](#yeah-but-how-do-i-use-that-asset-contract)
+    - [Native XLM on Soroban](#native-xlm-on-soroban)
+    - [Back to Your Quest](#back-to-your-quest)
```

Comment:
by convention, markdown likes to indent sub-bullet points by 2 spaces. maybe you've got a linter that changed this automatically?

Reply:
Oh nice catch!